x-pack/metricbeat/module/oracle/performance/metricset_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+// +build integration
+
+package performance
+
+import (
+	"testing"
+
+	"github.com/elastic/beats/x-pack/metricbeat/module/oracle"
+
+	"github.com/stretchr/testify/assert"
+
+	_ "gopkg.in/goracle.v2"
+
+	mbtest "github.com/elastic/beats/metricbeat/mb/testing"
+)
+
+func TestData(t *testing.T) {
+	t.Skip("Skip until a proper Docker image is setup for Metricbeat")
+	f := mbtest.NewReportingMetricSetV2WithContext(t, getConfig())
+
+	events, errs := mbtest.ReportingFetchV2WithContext(f)
```

Comment:
This and following checks are not needed as it is also checked by the `WriteEvents` methods.

Reply:
Sounds good! Done!